<p>This rule detects object allocation within the onDraw method of a class which extends a View Android component</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
