<p>This rule detects when new objects are allocated along with draw operations, which are very sensitive to performance.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
