<p>This rule detects an extractable method call which which receives no arguments, and is accessed by an object that is not altered in any way inside the loop.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
