<p>This rule detects the usage of the HashMap collection. The usage of HashMap is discouraged, since the alternative ArrayMap is allegedly more energy-efficient, without decreasing the performance of map operations</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
