<p>This rule detects the issue of having a non-static method inside a class, and which could be static instead, i.e., it does not access any class fields, it does not directly invoke non-static methods, and it is not an overriding method.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
