<p>This rule detects when collections or database related objects, such as TypedArrays or Cursors, are not recycled nor closed after being used.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
