<p>This rule detects failure to reuse data from already drawn items, which increases the number of calls to findViewById(), known to be energy greedy</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
